libkpod/stop.go

```diff
@@ -0,0 +1,36 @@
+package libkpod
+
+import (
+	"fmt"
+
+	"github.com/kubernetes-incubator/cri-o/oci"
+	"github.com/sirupsen/logrus"
+	pb "k8s.io/kubernetes/pkg/kubelet/api/v1alpha1/runtime"
+)
+
+// Stop stops a running container with a grace period (i.e., timeout).
+func (c *ContainerServer) Stop(container string, timeout int64) (string, error) {
```

Comment:
See above - can we remove the duplicate implementation in Server and call into this as well?